pyo3-ffi/src/genericaliasobject.rs

```diff
@@ -0,0 +1,21 @@
+use crate::object::{PyObject, PyTypeObject, Py_TYPE};
+use crate::PyObject_TypeCheck;
+use std::os::raw::c_int;
+use std::ptr;
+
+#[inline]
+pub unsafe fn PyGenericAlias_CheckExact(op: *mut PyObject) -> c_int {
+    (Py_TYPE(op) == ptr::addr_of_mut!(Py_GenericAliasType)) as c_int
+}
+
+#[inline]
+pub unsafe fn PyGenericAlias_Check(op: *mut PyObject) -> c_int {
+    PyObject_TypeCheck(op, ptr::addr_of_mut!(Py_GenericAliasType))
+}
+
+#[cfg_attr(windows, link(name = "pythonXY"))]
+extern "C" {
+    pub fn Py_GenericAlias(origin: *mut PyObject, args: *mut PyObject) -> *mut PyObject;
```

Comment:
```suggestion
    #[cfg(Py_3_9)]
    pub fn Py_GenericAlias(origin: *mut PyObject, args: *mut PyObject) -> *mut PyObject;
```

and below as well for Py_GenericAliasType` because we still support Python 3.8 and these symbols aren't present in the 3.8 headers.

Reply:
Done